github-actions/oss-readme-template/__tests__/readme.test.ts

```diff
@@ -213,10 +223,88 @@ This is an official Momento SDK, but the API is in a beta stage.  For more info,
         projectStability: ProjectStability.STABLE,
       })
     ).toThrowError(
-      /Expected to find next header with content 'Requirements', found 'FOO'/
+      /After header 'Getting Started :running:', missing expected header: 'Requirements'/
     );
   });
 
+  it('succeeds for an SDK README that includes all of the expected section headers, but also includes additional headers', () => {
+    const templateContents = fs
+      .readFileSync(
+        path.join(
+          __dirname,
+          'workflows',
+          'valid-sdk-template-with-extra-headers.md'
+        )
+      )
+      .toString();
+
+    expect(
+      generateReadmeStringFromTemplateString({
+        templateContents: templateContents,
+        projectInfo: EXAMPLE_SDK_PROJECT_INFO,
+        projectStatus: ProjectStatus.OFFICIAL,
+        projectStability: ProjectStability.STABLE,
+      })
+    ).toEqual(`<head>
+  <meta name="Momento WaterLoop Client Library Documentation" content="WaterLoop client software development kit for Momento Serverless Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md) 
+
+# Momento WaterLoop Client Library
+
+
+WaterLoop client SDK for Momento Serverless Cache: a fast, simple, pay-as-you-go caching solution without
+any of the operational overhead required by traditional caching solutions!
```

Comment:
Not a blocker for the PR but would help maintainability to externalize expected generated output to separate files

Reply:
yeah that's a good point, next time i'm in here I'll improve it.